### ISSUE TITLE
Support for several expiration times?

### ISSUE THREAD

**Issue by iconiczz** [nokonoko/Uguu#108]
Hi, is it possible for a feature to be added to have different expiration times set up? It would be nice if it could be set to like a few hours, days and years.

**Comment by nokonoko** [comment]
Uguu had support for this feature several years ago, but I removed it due to the added friction for the user.

It goes against Uguu's (Pomf's) design philosophy, which is to just upload a file without paying, making an account, modify settings etc.

However adding it as just an option via the API would be fine, might look into that for next release.

**Comment by iconiczz** [comment]
> However adding it as just an option via the API would be fine, might look into that for next release.

Yes that would be nice. Is it possible as well for it to be enabled by the host, making it work on the website so that if I were looking to self host this as part of my personal tools on a raspberry pi or something I could enable it?

Thanks!

**Comment by nokonoko** [comment]
You can set different expire times in the configuration file, however a way to set expire times upon upload from the UI/API will not be added at this time, maybe in the future, maybe not.
